src/simulations/modeling/cobra_helpers.py

```diff
@@ -159,3 +163,41 @@ def _query_item(item, query_id, query_namespace):
                 if query_id.lower() == annotation.lower():
                     return True
     return False
+
+
+def parse_bigg_compartment(metabolite_id, model):
+    """
+    Parse the compartment ID of the given metabolite BiGG identifier.
```

Comment:
```suggestion
    Parse the compartment ID of the given metabolite identifier.
```

Reply:
Why not be explicit that we're expecting bigg convention here?

Reply:
Just wanted to express that we're not actually looking for BiGG identifiers only but it could be any identifier, e.g., MNXM, that we expect to have a compartment suffix. However, that turned out to be too long for the summary line :stuck_out_tongue:

Reply:
Ahh... I thought the compartment postfix was purely a bigg convention (and for example metanetx used `MNXM1@MNXC1`). Should probably rename the function too then!

Reply:
It is! To be more specific, I did not want to give the impression that the function only works on BiGG identifiers (the set of semi-understandable symbols) but any kind of identifier that has potentially an underscore compartment suffix.